apps/betterangels/src/app/(private-screens)/add-note/[noteId].tsx

```diff
@@ -1,6 +1,5 @@
 import {
```

Comment:
Hey @Davit-BetterAngels 

I found a couple of things.

1. The search box doesn't have the x button to clear the text field
2. When the search box is used with a keyword, the search result e.g., tent remains even though I close the Services List with or without being checked. When the drawer closed, the whole list should be back if the search box is cleared/empty. And the checkbox should be checked if the user is already chosen the items. In another word, how do I see the whole list after I searched a keyword and successfully selected the item? 
3. When I tapped Done button twice accidentally, the items are shown twice on Provided Services/Requested Services. Maybe we can use a visual aid = loading animation to let the user know it's being processed? 


**No. 1 and 2**
<img src="https://github.com/user-attachments/assets/ee6986ca-3a86-4fde-82d6-813d666f0c02" width="360" />

**No. 3**
<img src="https://github.com/user-attachments/assets/95acc9dc-7619-4776-ad08-e5f5094d7da7" width="360" />


Let me know your thought and your questions if you have any.
Thank you!

Reply:
We might have to create a ticket for No2. But let me know.